doc/source/themes/scikit-image/static/css/custom.css

```diff
@@ -298,3 +298,15 @@ div.sphx-glr-download a:hover {
 p.sphx-glr-signature a.reference.external {
   display: none !important;
 }
+
+a.copybtn {
+    background: transparent no-repeat none;
+    border:none;
+    position: absolute;
+    top: 0;
+    right: 0;
+    margin: 0.5px !important;
```

Comment:
Why is the `!important` necessary here?

Reply:
I am not familiar with how sphinx and its extensions work. But  my css style won't change if I don't use `!important`. `sphinx-gallery` extension also used `!important` to enforce its style in `custom.css`.